features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

```diff
@@ -0,0 +1,204 @@
+
+        $.template("policy-list", policyListingSrc, function (template) {
+            var serviceURL = "/devicemgt_admin/policies";
+
+            var successCallback = function (data) {
+                data = JSON.parse(data);
+                $('#policy-spinner').addClass('hidden');
+                var policyListFromRestEndpoint = data.responseContent;
+                if (policyListFromRestEndpoint.length > 0) {
+                    var viewModel = {};
+                    var policyListToView = [];
+                    var i, policyObjectFromRestEndpoint, policyObjectToView;
+                    for (i = 0; i < policyListFromRestEndpoint.length; i++) {
+                        // get list object
+                        policyObjectFromRestEndpoint = policyListFromRestEndpoint[i];
+                        // populate list object values to view-object
+                        policyObjectToView = {};
+                        policyObjectToView["id"] = policyObjectFromRestEndpoint["id"];
+                        policyObjectToView["priorityId"] = policyObjectFromRestEndpoint["priorityId"];
+                        policyObjectToView["name"] = policyObjectFromRestEndpoint["policyName"];
+                        policyObjectToView["platform"] = policyObjectFromRestEndpoint["profile"]["deviceType"]["name"];
+                        policyObjectToView["ownershipType"] = policyObjectFromRestEndpoint["ownershipType"];
+                        policyObjectToView["compliance"] = policyObjectFromRestEndpoint["compliance"];
+
+                        if (policyObjectFromRestEndpoint["active"] == true && policyObjectFromRestEndpoint["updated"] == true) {
+                            policyObjectToView["status"] = "Active/Updated";
+                        } else if (policyObjectFromRestEndpoint["active"] == true && policyObjectFromRestEndpoint["updated"] == false) {
+                            policyObjectToView["status"] = "Active";
+                        } else if (policyObjectFromRestEndpoint["active"] == false && policyObjectFromRestEndpoint["updated"] == true) {
+                            policyObjectToView["status"] = "Inactive/Updated";
+                        } else if (policyObjectFromRestEndpoint["active"] == false && policyObjectFromRestEndpoint["updated"] == false) {
+                            policyObjectToView["status"] = "Inactive";
+                        }
+                        // push view-objects to list
+                        policyListToView.push(policyObjectToView);
+                    }
+                    viewModel.policies = policyListToView;
+                    var content = template(viewModel);
+                    $("#policy-list-container").html(content);
+                    $('#policy-table').datatables_extended();
+                }
+            };
+            invokerUtil.get(serviceURL,
+                            successCallback, function (message) {
+                        console.log(message);
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/303/commits/051493db6a3fdb2e75cc7d1f48cb3ed407622312